### ISSUE TITLE
Wav2vec 2.0 Accuracy

### ISSUE THREAD

**Issue by sooftware** [pytorch/fairseq#2643]
## ❓ Questions and Help

I'm pre-training the wav2vec 2.0 model.   
In tensorboard currently, accuracy approximately 0.3.  
  
I wonder if this 0.3 is 30 percent or 0.3 percent.
  
Thank you FAIR for open this great paper and code.
